docs/source/api/v2/staticdnsentries.rst

```diff
@@ -314,7 +314,7 @@ Response Structure
 		"lastUpdated": "2018-12-10 19:59:56+00",
 		"ttl": 300,
 		"type": null,
```

Comment:
can. you fix type to not be null? i think type is supposed to be `type: CNAME_RECORD`

Reply:
Code updated.